operatorcourier/cli.py

```diff
@@ -55,11 +55,17 @@ def push(self):
         args, leftovers = parser.parse_known_args(sys.argv[2:])
         api.build_verify_and_push(args.namespace, args.repository, args.release, args.token, source_dir=args.source_dir)
 
-    # Parse the push command
+    # Parse the nest command
     def nest(self):
         parser = argparse.ArgumentParser(description='Take a flat bundle directory and version nest it to eventually create an operator-registry image.')
         parser.add_argument('source_dir', help='Path of your directory of yaml files to bundle.')
         parser.add_argument('registry_dir', help='Path of your directory to be populated. If directory does not exist, it will be created.')
 
         args, leftovers = parser.parse_known_args(sys.argv[2:])
         api.nest(args.source_dir, args.registry_dir)
+
+    #Parse the version command
+    def version(self):
```

Comment:
I fail to see where the version number is printed to screen?

Reply:
Same. I don't think this pr actually implements the command.

Reply:
@Avni-Sharma could you please print the version number to the screen?